src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -121,6 +121,33 @@ The features below are only available at the page level. (You can't use them ins
 This is because these features modify the [Response headers](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), which can't be modified after they're sent to the browser. In SSR mode, Astro uses HTML streaming to send each component to the browser as it renders them. This makes sure the user sees your HTML as fast as possible, but it also means that by the time Astro runs your component code, it has already sent the Response headers.
 :::
 
+### `Astro.cookies`
+
+This is a utility to read and modify a single cookie. It allows you to check, set, get and delete a cookie.
+
+Use `Astro.request.headers.get('cookie')` to get all cookies at once.
```

Comment:
This gets the cookie string header. Given there isn't an explanation for this or why you might want it, should this sentence be here?

Reply:
But it's from the example right before, without any mention people might be confused what the difference between them are. If that is not a concern, then can be dropped for simplicity

Reply:
I think the fact that we've emphasized a *single* cookie makes this clear enough without the extra statement?